packs/world-altar/data/world-altar/functions/sacrifice/diamond.mcfunction

```diff
@@ -1,6 +1,6 @@
 setblock ~ ~2 ~ minecraft:air
 playsound block.beacon.activate master @a ~ ~ ~ 100
 playsound entity.lightning_bolt.thunder master @a ~ ~ ~ 100
-worldborder add 137
-tellraw @a ["",{"text":"\u26a1 A ","bold":true},{"text":"diamond","bold":true,"color":"aqua"},{"text":" block was sacrificed","bold":true},{"text":" \u26a1","bold":true},{"text":"\n"},{"text":"The world border has been expanded by ","color":"gray"},{"text":"137","bold":true,"color":"white"},{"text":" blocks.","color":"gray"},{"text":"\n"},{"selector":"@p","color":"light_purple"},{"text":" was the closest player.","color":"gray"}]
+worldborder add 20
```

Comment:
Ha I think this might be slightly too much of a nerf, I'm gonna approve anyway but I'd maybe be happier with diamond = 50. At 20, to get to the end portal will take 21 full stacks of diamonds 😁

A 50 block change will cost us 8 full stacks which seems a bit more achievable